### PR TITLE
Update SIG Platform README

### PR DESCRIPTION
**Pull request by LaVLaS**
Updates SIG Platform README to include meeting schedule and member github profiles

### CODE REVIEW

File: sig-platform/README.md [hunk 1]
```diff
@@ -5,26 +5,26 @@ SIG Platform covers the ODH operator, odh-core components, ecosystem integration
 The [charter](./charter.md) defines the scope and governance of the Platform Special Interest Group.
 
 ## Meetings
-[Monthly SIG meeting:] TBD
-[Meeting notes and agenda] RBD
+- Bi-Wonthly SIG meeting: Second and Fourth Mondays @ 11am Eastern Time ([Meeting Link](https://meet.google.com/vyw-gypz-siz))
```
Comment (VaishnaviHire) [inline sig-platform/README.md:8]: Small typo
```
Bi-Wonthly --> Bi-Monthly
```

**Review by VaishnaviHire — APPROVED** [review]
/lgtm

**Review by taneem-ibrahim — APPROVED** [review]
/lgtm

**Pull request merged**